src/stream/src/executor/simple_agg.rs

```diff
@@ -64,6 +64,9 @@ struct ExecutorInner<S: StateStore> {
     /// An operator will support multiple aggregation calls.
     agg_calls: Vec<AggCall>,
```

Comment:
Is `agg_calls` still needed after we have `agg_funcs` field?

Reply:
In fact I'm working on a subsequent refactor to merge `agg_calls`, `agg_storages`, `agg_funcs` and other variables into one field, which is something equivalent of `AggregateFunction` in executors.